resolwe_bio/utils/test.py

```diff
@@ -15,18 +15,19 @@
 )
 
 
-def skipDockerFailure(reason):
-    """Skip the decorated test unless ``TESTS_SKIP_DOCKER_FAILURES``
-    Django setting is set to ``False``. ``reason`` should describe why
-    the test is being skipped.
+def skipDockerFailure(reason):  # pylint: disable=invalid-name
+    """Skip the decorated tests.
+
+    Unless ``TESTS_SKIP_DOCKER_FAILURES`` Django setting is set to
+    ``False``. ``reason`` should describe why the test is being skipped.
     """
     if getattr(settings, 'TESTS_SKIP_DOCKER_FAILURES', True):
         return unittest.skip(reason)
     return lambda func: func
 
 
-def skipUnlessLargeFiles(*files):
-    """Skip the decoreted tests unless the given large files are available.
+def skipUnlessLargeFiles(*files):  # pylint: disable=invalid-name
+    r"""Skip the decoreted tests unless a large file is available.
```

Comment:
We have to escape the * in the param docstring below. I had \\ before, but Pylint warned me not to use double backslashes and suggested that I should use the r" syntax.